index.html

```diff
@@ -144,57 +128,84 @@ <h3>Multi-Post Stories Gain+Glory</h3>
+            don’t hestiate to contact me.
+          </p>
 
-        <button type="submit" class="resume-button">Get my Resume</button>
-      </article>
+          <button type="submit" class="resume-button">Get my Resume</button>
+        </article>
 
         <article class="card-about-me">
-         <div class="icon-holder"><img src="imgs/lang-icon.png" alt="lang-icon" width="48" height="48"></div>
-         <h3>Languages</h3>
-         <ul>
-          <li>JavaScript</li>
-          <li>Ruby</li>
-          <li>Html</li>
-          <li>CSS</li>
-         </ul>
+          <div class="icon-holder"><img src="imgs/lang-icon.png" alt="lang-icon" width="48" height="48"></div>
+          <h3>Languages</h3>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
         </article>
        
         <article class="card-about-me">
           <div class="icon-holder"><img src="imgs/frameworks-icon.png" alt="frameworks-icon" width="48" height="48"></div>
-         <h3>Frameworks</h3>
-         <ul>
-          <li>Bootstrap</li>
-          <li>Ruby on rails</li>
-          <li>RSpec</li>
-          <li>Capybara</li>
-          <li>Selenium</li>
-         </ul>
+            <h3>Frameworks</h3>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
         </article>
         
         <article class="card-about-me">
-          <div></div><div class="icon-holder"><img src="imgs/skills-icon.png" alt="skills-icon" width="48" height="48"></div>
+          <div class="icon-holder"><img src="imgs/skills-icon.png" alt="skills-icon" width="48" height="48"></div>
           <h3>Skills</h3>
           <ul>
-           <li>Codekit</li>
-           <li>GitHub</li>
-           <li>Codepen</li>
-           <li>Gitlab</li>
-           <li>Terminal</li>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
           </ul>
         </article>
+      </section>
+
+      <section class="contact-section">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
 
+        <form action="https://formspree.io/f/xvolveaz" method="post">
+          
+          <ul>
+            <li><input type="text" name="first_name" placeholder="Full name" required maxlength="30"/></li>
+            <li><input type="email" name="email" placeholder="Email address" required/></li>
+            <li>
+            <textarea required maxlength="500">Enter text here</textarea>
+            </li>
+            <li><button type="submit">Get in touch</button></li>
```

Comment:
Kindly implement button interactions as shown in the Figma design 🤗

![image](https://user-images.githubusercontent.com/73714615/187037725-59968609-8ac8-4368-ae9d-3c5e1ed76092.png)

Reply:
implemented.